CONTRIBUTING.md

```diff
@@ -14,78 +14,127 @@ See the License for the specific language governing permissions and
+The [Getting Started Quick Start Guide](contributing/getting-started-quick.md) has installation steps and instructions for building/testing AMP.
 
-**Google Individual Contributor License**
+[DEVELOPING.md](DEVELOPING.md) has some more advanced instructions that may be necessary depending on the complexity of the changes you are making.
 
-Code contributors to the AMP HTML project must sign a Contributor License Agreement, either for an [individual](https://developers.google.com/open-source/cla/individual) or [corporation](https://developers.google.com/open-source/cla/corporate). The CLA is meant to protect contributors, users of the AMP HTML runtime, and Google in issues of intellectual property.
+A few things to note:
 
-### Contributing Features
+* The AMP Project follows the [fork & pull](https://help.github.com/articles/using-pull-requests/#fork--pull) model for accepting contributions.
+* Familiarize yourself with our [Design Principles](DESIGN_PRINCIPLES.md).
+* We follow [Google's JavaScript Style Guide](https://google.github.io/styleguide/jsguide.html).  More generally make sure to follow the same comment and coding style as the rest of the project.
+* Include tests when contributing code.  There are plenty of tests that you can use as examples.
+* A key feature of AMP is performance.  All changes will be analyzed for any performance impact; we particularly appreciate changes that make things even faster.  Please include any measured performance impact with substantial pull requests.
 
-All pull requests for new features must go through the following process:
-* Please familiarize yourself with the [AMP Design Principles](DESIGN_PRINCIPLES.md)
-* Start an Intent-to-implement GitHub issue for discussion of the new feature.
-* LGTM from Tech Lead and one other core committer is required
-* Development occurs on a separate branch of a separate fork, noted in the intent-to-implement issue
-* A pull request is created, referencing the issue.
-* AMP HTML developers will provide feedback on pull requests, looking at code quality, style, tests, performance, and directional alignment with the goals of the project. That feedback should be discussed and incorporated
-* LGTM from Tech Lead and one other core committer, who confirm engineering quality and direction.
+## Contributing features
 
-#### Contributing Extended Components
+Follow this process for contributing new features:
+* Familiarize yourself with the [AMP Design Principles](DESIGN_PRINCIPLES.md)
+* [Create a new GitHub issue](https://github.com/ampproject/amphtml/issues/new) with the [Intent to Implement label](https://github.com/ampproject/amphtml/labels/INTENT%20TO%20IMPLEMENT) to start discussion of the new feature.
+* Consider bringing the eng design for your feature to our [weekly design review](#weekly-design-review).
+* Before starting on the code for your feature, get approval from the [Tech Lead and a core contributor](GOVERNANCE.md).
```

Comment:
Changed the link to specifically point to the core committers section.  I also changed TL to OWNER since I think that's the intent now?

Was your suggestion to move most of the bullet points you added in GOVERNANCE to here?